CMakeLists.txt

```diff
@@ -153,6 +153,8 @@ if(WIN32)
     set(OTIO_DEBUG_POSTFIX "d")
 endif()
 
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
```

Comment:
just googled what this does, neat.

Reply:
It definitely helps keep things neat in IDEs like Visual Studio, though one downside is you need to manually specify the folders for each target:
`set_target_properties(${example} PROPERTIES FOLDER examples)`